Automattic/sensei
# Add 'Completed' and 'Next Lesson' button to Lesson Actions

### Changes proposed in this Pull Request

- When the lesson is completed, display a disabled Completed button in place of the Complete button. Previously looking at an already completed lesson felt a bit awkward as it wasn't always clear why there are no actions available.

<img width="297" alt="image" src="https://user-images.githubusercontent.com/176949/192488726-74627179-28fd-44d2-8416-8f46ca30059d.png">

- Also display a 'Next Lesson' link next to this button, to have a primary action. I added an attribute to make this optional, so it won't show up for the legacy template, which already has the previous/next links.

- Update the lesson actions in the bottom of the default template to behave like this

### Testing instructions

* Set the default template for LM
* View a lesson, and complete it
* Go back to the completed lesson
* 'Completed button' should show in place of the Complete action
* 'Next Lesson' link should also show up in the bottom action area (unless it's the last lesson in the course)
